selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -106,27 +106,34 @@ void Networking::requestScan() {
   wifi->requestScan();
 }
 
-void Networking::refreshSlot() {
-  wifiWidget->refresh();
-  an->refresh();
+void Networking::refresh() {
+  if (this->isVisible() || wifi->firstScan) {
+    wifiWidget->refresh();
+    an->refresh();
+    wifi->firstScan = false;
```

Comment:
why is this in wifi? only used in networking

Reply:
We need to allow a refresh once in the background. If we get rid of `isVisible` NM can sometimes send a scan signal and the UI will update in the background at any time which isn't desired. And we can't use `ui_setup_complete` because it will be true already when we're here (since we're only here if the scan is complete)

Reply:
In the future when UI updating is quick, we can remove this and only update when the view is tapped